{
    "original_source": {
        "database(url)": "https://github.com/secureIT-project/CVEfixes",
        "software_system(CPE)": [
            "cpe:2.3:a:calibre-web_project:calibre-web:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/janeczku/calibre-web/commit/4545f4a20d9ff90b99bbd4e3e34b6de4441d6367",
        "version_end_excluding": {
            "calibre-web_project:calibre-web": "0.6.18"
        },
        "path": {
            "file_path": "cps/__init__.py"
        }
    },
    "programming_language": "python",
    "licences": {
        "database": "MIT",
        "affected_repository": "GNU General Public License v3.0"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2022-0939"
        ],
        "CWE": [
            "CWE-918"
        ]
    }
}